lighthouse-core/report/v2/report-styles.css

```diff
@@ -739,6 +745,7 @@ summary.lh-passed-audits-summary::-webkit-details-marker,
 @media screen and (max-width: 767px) {
   .lh-report {
     margin-left: 0;
+    width: 100%;
```

Comment:
added min-width 400px which is enough to cover the header gauges to prevent them from being their own separate scroll box, can probably lower it after #2297 :)